TODO

```diff
@@ -32,6 +32,11 @@ Janitorial Clean-ups:
 
 Features:
 
+* drop nss-myhostname in favour of nss-resolve?
+
```

Comment:
well i added the question mark to this entry, since I am not entirely sure about it either...